examples/asset/utilization/AssetUtilization.cs

```diff
@@ -0,0 +1,47 @@
+﻿using NationalInstruments.SystemLink.Clients.AssetManagement;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.Asset.Utilization
+{
+    /// <summary>
+    /// Example for the SystemLink Asset Management client API that
+    /// tracks utilization for assets present in the system.
+    /// 
+    /// Note: As a prerequisite for this example, ensure SystemLink Client is installed.
+    /// </summary>
+    class AssetUtilization
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * The Asset Utilization Store enables asset utilization tracking. 
+             * It is created using a factory.
+             */
+            var utilizationStoreFactory = new AssetUtilizationStoreFactory();
+            var assetUtilizationStore = utilizationStoreFactory.CreateAssetUtilizationStore();
+
+            /*
+             * Tracking Asset Utilization can be configured to send heartbeats
```

Comment:
"Track asset utilization by configuring the API to send heartbeats for ongoing  sessions at predefined intervals."?